src/main/webapp/go.js

```diff
@@ -45,12 +47,14 @@ window.onload = function() {
   getHunt();
 };
 
+window.setInterval(updateGeolocation, REFRESH_TIME);
```

Comment:
I think it would be better to do this inside the createMap method to make sure it won't fire before the maps API is ready